draft-ietf-acme-acme.md

```diff
@@ -2639,6 +2633,10 @@ particular case.  For identifiers where the server already has some public key
 associated with the domain this attack can be prevented by requiring the client
 to prove control of the corresponding private key.
 
+It is RECOMMENDED that the server perform DNS queries and make HTTP and TLS
+connections from various network perspectives, in order to make MitM attacks
```

Comment:
Does it make sense to reference this is talking about validations? I.e. `DNS queries and make HTTP and TLS connections use for validation`